internal/cli/performanceadvisor/suggestedindexes/list.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package suggestedindexes
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const listTemplate = `ID	NAMESPACE	WEIGHT{{range .SuggestedIndexes}}
```

Comment:
[q] have you managed to run this command? I ask as I'm trying to play with it and trying to get some recommendations for indices.
Also related and big thing here, this output is not helpful for users, similar to the logs one we need to actually show the index recommendation, I'm aware that is a struct so it would be nice to see how that struct can be rendered inline

Reply:
yes, I did run it on my atlas cluster with the Airbnb database but I keep getting an empty string. I am not sure how to get suggested indexes though since the Performance Advisor monitors queries that take longer than 100 milliseconds and this Airbnb db may not be big enough (https://docs.opsmanager.mongodb.com/current/performance-advisor/index-ranking/#how-pa-suggests-and-ranks-indexes). Do you have any suggestions?